x/leverage/keeper/collateral.go

```diff
@@ -77,3 +77,10 @@ func (k Keeper) GetBorrowerCollateral(ctx sdk.Context, borrowerAddr sdk.AccAddre
 	totalCollateral.Sort()
 	return totalCollateral
 }
+
+// GetEligibleLiquidationTargets returns a list of borrower addresses eligible for liquidation
+func (k Keeper) GetEligibleLiquidationTargets(ctx sdk.Context) ([]sdk.AccAddress, error) {
```

Comment:
Do we actually need this to be defined in the state-machine? Can the tool not just query on-chain state to figure out the targets?

Granted, we could just build the query for them too...